apel/common/message_schemas.py

```diff
@@ -0,0 +1,149 @@
+"""A file to store message schemas for JSON based messages"""
+
+IP_MSG_SCHEMA = {
+    "definitions": {
+
+    },
+    "$schema": "http: //json-schema.org/draft-07/schema#",
+    "$id": "http: //example.com/root.json",
+    "type": "object",
+    "title": "The Root Schema",
+    "required": [],
```

Comment:
We should probably explicitly express that these fields are required, as if any of them are missing the message wont/shouldn't be parsed properly.

```suggestion
    "required": [
        "Type",
        "Version",
        "UsageRecords"
    ],
```